corehq/apps/domain/views.py

```diff
@@ -1439,25 +1464,52 @@ def selected_plan_version(self):
         return DefaultProductPlan.get_default_plan_version(self.edition)
 
     @property
-    def downgrade_messages(self):
-        subscription = Subscription.get_active_subscription_by_domain(self.domain)
-        downgrades = get_change_status(
-            subscription.plan_version if subscription else None,
-            self.selected_plan_version
-        )[1]
-        downgrade_handler = DomainDowngradeStatusHandler(
-            self.domain_object, self.selected_plan_version, downgrades,
-        )
-        return downgrade_handler.get_response()
+    def is_upgrade(self):
```

Comment:
What if ```current_edition``` == ```next_edition```? ```is_upgrade``` would imply this should return ```False```, but I think it will return ```True```.

Reply:
Thoughts on this?  It's ok if the answer is "keep as-is", as long as they're a reason for it.

Reply:
Whoops, I must have skipped over this! You're right that it should return False. I'll update this now

Reply:
What name were you thinking?

Reply:
Actually this makes sense.  For some reason I was confused earlier about what happens when current_edition == next_edition